packages/checkbox/src/css/index.js

```diff
@@ -62,26 +62,14 @@ export default {
     right: '-7px',
     bottom: '-7px'
   },
-  '.psds-checkbox__square:focus:before': {
```

Comment:
what were these?  why can they all go away?

seems related to focus rings.  But in that case, why are there still so many lines of css for focus rings remaining?

Reply:
this was missed when we did the focus ring to halo component refactor. there is no :before element to target